enterprise/server/backends/migration_cache/config.go

```diff
@@ -21,6 +21,9 @@ type MigrationConfig struct {
 	// CopyChanFullWarningIntervalMin controls how often we should log when the copy chan is full
 	CopyChanFullWarningIntervalMin int64 `yaml:"copy_chan_full_warning_interval_min"`
 	MaxCopiesPerSec                int   `yaml:"max_copies_per_sec"`
+	// Whether we rate limit Write(). If so, we will write to destination cache
+	// in the background and use MaxCopiesPerSec to control the speed.
+	RateLimitWriteToDest bool `yaml:"rate_limit_writes"`
```

Comment:
nit: i'd call this something like `async_dest_writes` since the rate limiting is only one of the effects of the change.